packages/select-rich/src/LionSelectRich.js

```diff
@@ -47,6 +47,10 @@ export class LionSelectRich extends SlotMixin(ScopedElementsMixin(OverlayMixin(L
         reflect: true,
         attribute: 'single-option',
       },
+      arrowWidth: {
+        type: Number,
+        attribute: 'arrow-width',
+      },
```

Comment:
```suggestion
      arrowWidth: {
        attribute: false,
      },
```
Since there is no purpose for linking to the attribute as far as I can tell. Unless you think arrow-width should be passed by the app dev declaratively? But it's more of a subclasser thing to take care of

Reply:
Actually why is it a Lit property to begin with? It doesn't seem to respond to imperative changes of the property. Unless the idea is to add an updated hook and call alignInvokerWidth when arrowWidth changes?